person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] **RECOMMENDED:** Kindly make sure you move the `of_age?` method to come after the `can_use_services?` method. In the way you define them now, all are private instead whereas  the `of_age?` method is the only private and the other one is `public` i.e.:
```ruby

class Person
 ...

 def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >= 18
  end
end